api/postman/bookings-postman.json

```diff
@@ -0,0 +1,1983 @@
+{
```

Comment:
I'm guessing that this file shouldn't be part of this commit / PR?

Reply:
No I updated the file to include the new field I added.